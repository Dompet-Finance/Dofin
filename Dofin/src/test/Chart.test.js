import 'react-native';
import 'react-dom';
import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

describe("unit testing for chart", () => {

  test("it should render some data", () => {

    class SimpleChart extends React.Component {
      constructor(props){
        super(props)
        this.state = {
          jml_category: 10
        }
      }
      render() {
        return <div>{this.state.jml_category}</div>;
      }
    }
    const wrapper = renderer.create(<SimpleChart />);
    expect(wrapper).toMatchSnapshot()
  })

})
