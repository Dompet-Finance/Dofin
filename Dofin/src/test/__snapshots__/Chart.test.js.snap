// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`unit testing for chart it should render some data 1`] = `
<div>
  10
</div>
`;
